packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.class.js

```diff
@@ -0,0 +1,49 @@
+import get from 'lodash/get';
+
+import {
+  BACKUP_STATE_DISABLED,
+  BACKUP_OFFER_LEGACY,
+  BACKUP_OFFER_ADVANCED,
+  BACKUP_OFFER_PREMIUM,
+} from './backup.constants';
+
+export default class Backup {
+  constructor(backup) {
+    Object.assign(this, backup);
+    this.initializeBackup();
+  }
+
+  initializeBackup() {
+    this.backupSizeInReport = this.backupSizeInReport
+      ? this.backupSizeInReport
+      : false;
+    this.fullDayInReport = this.fullDayInReport ? this.fullDayInReport : false;
+    this.restorePointInReport = this.restorePointInReport
+      ? this.restorePointInReport
+      : false;
+    this.diskSizeInReport = this.diskSizeInReport
+      ? this.diskSizeInReport
+      : false;
+    this.encryption = this.encryption ? this.encryption : false;
+    this.backupDurationInReport = this.backupDurationInReport
+      ? this.backupDurationInReport
+      : false;
+    this.sendReportToCustomeEmail = this.mailAddress !== null;
```

Comment:
Maybe use a getter for this 
```suggestion
   get sendReportToCustomerEmail() {
     return this.mailAddress !== null
    }
```

Reply:
This is done